modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * security/tls-profiles.adoc
+
+[id="tls-profiles-kubernetes-configuring_{context}"]
+= Configuring the TLS security profile for the Kubernetes control plane
+
+To configure a TLS security profile for the Kubernetes control plane, edit the `APIServer` custom resource (CR).  If a TLS profile is not configured, the default TLS profile is `Intermediate`. The Kubernetes control plane consists of the Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager.
```

Comment:
The control plane technically consists of more components, so need to rework that last sentence a little. Maybe:

```suggestion
To configure a TLS security profile for the Kubernetes control plane components (Kubernetes API server, the Kubernetes scheduler, and the Kubernetes controller manager), edit the `APIServer` custom resource (CR). If a TLS profile is not configured, the default TLS profile is `Intermediate`. 
```